src/selectors/index.js

```diff
@@ -1,3 +1,5 @@
+import { meetsDuringDow, meetsDuringHour } from 'components/calendar/calendar-helpers';
```

Comment:
Now that these are being used outside of components, let's put these in `util/time` instead and import them from there.